### ISSUE TITLE
k8s custom dns domain support

### ISSUE THREAD

**Issue by qianlongzt** [telepresenceio/telepresence#1785]
cluser init with custom dns domain, but telepresence don't know it 
``` yaml
  networking:
      dnsDomain: k8s
```

cat /etc/resolver/telepresence.local
```
# Generated by telepresence
port 52733
domain cluster.local
nameserver 127.0.0.1
search cluster.local
```

but it should be

```
# Generated by telepresence
port 52733
domain k8s
nameserver 127.0.0.1
search  k8s
```

**Comment by qianlongzt** [comment]
dns work for me https://www.telepresence.io/docs/latest/reference/config/#dns


```yaml
        dns:
          include-suffixes:
          - .k8s
```
but when i quit cluster, file `/etc/resolver/telepresence.k8s.local` not auto delete

**Comment by efunk** [comment]
Resolved in Telepresence 2.3.2 GA

**Comment by maelvls** [comment]
-> moved my question to its own issue: https://github.com/telepresenceio/telepresence/issues/1943
